### ISSUE TITLE
Number of typos on a page

### ISSUE THREAD

**Issue by TLMurray23** [MicrosoftDocs/feedback#787]
**Link to article:**
https://docs.microsoft.com/en-us/windows/desktop/FileIO/naming-a-file#file-and-directory-names

**Problem:**
Generally a number of typos, usually in the area of missing characters. Just need some basic editing.

**Comment by Powerhelmsman** [comment]
@ginami Please take a look at this issue.

**Comment by TLMurray23** [comment]
No, the typos are still there. Look at the second bullet under Naming Conventions and the missing greater than, a bit further down.

**Comment by ginami** [comment]
Thanks @TLMurray23 -- I'll find someone to update the page!

**Comment by nschonni** [comment]
TLooks like a duplicate of #733, but that's not resolved either
